tests/utils/test_singleton.py

```diff
@@ -23,9 +23,19 @@ class A(metaclass=Singleton):
     pass
 
 
+class Counter(metaclass=Singleton):
```

Comment:
Probably needs a docstring or some better named class/variable names, I imagine I’d have difficulties understanding what this class (and the test) actually does if I read this code a couple of months later.

Reply:
Fixed